iceoryx2-ffi/cxx/tests/src/unique_port_id_tests.cpp

```diff
@@ -63,6 +65,13 @@ struct UniquePortIdTest : public ::testing::Test {
 
 TYPED_TEST_SUITE(UniquePortIdTest, iox2_testing::ServiceTypes);
 
+TYPED_TEST(UniquePortIdTest, unique_publisher_id_value) {
+    auto null_id = std::array<uint8_t, iox2::UNIQUE_PORT_ID_LENGTH> {};
```

Comment:
I initially wanted the Rust layer to just provide a pointer to the ID bytes, however I couldn't work out how to resolve the ownership between the two layers, so I opted for a copy instead.
Do you see any better options?